core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java

```diff
@@ -640,6 +655,18 @@ private void setOperationIdOnce(String val, boolean suppressError) {
     operationId = TabletOperationId.from(val);
   }
 
+  /**
+   * Sets the next operation ID only once.
+   *
+   * @param val operation id to set
+   * @param suppressError set to true to suppress an exception being thrown, else false
+   * @throws IllegalStateException if an operation id or location is already set
+   */
+  private void setNextOperationIdOnce(String val, boolean suppressError) {
+    Preconditions.checkState(nextOperationId == null);
```

Comment:
If its not expected that tablet should ever have an operation id and next operation id set, then could check that here and in setOperationIdOnce